app/src/docs/v1.api-spec.yaml

```diff
@@ -1085,6 +1128,37 @@ components:
           type: boolean
           description: User is an owner of this acronym
           example: false
+        createdAt:
+          type: string
+          description: The datestamp the user->acronym mapping was created
+          example: "2019-08-22T18:43:07.393Z"
+    User:
+      type: object
+      properties:
+        userId:
+          type: string
+          description: Primary key from the user table
+          example: cc745445-e221-4a9a-8758-2d7d20c1decf
+        keycloakGuid:
+          type: string
+          description: Identifier for the user in KC
+          example: 5b5d0a2f-93a0-4c18-9d70-753648b8e7b9
+        username:
+          type: string 
+          description: Username from KC
+          example: jsmith@idir
+        firstName:
+          type: string
+          description: First name from KC
+          example: John
+        lastName:
+          type: string
+          description: Last name from KC
+          example: Smith
+        email:
+          type: string
+          description: Email from KC
+          example: jsmith@gov.bc.ca
```

Comment:
Ideally should be alphabetical unless there is a logical reason on ordering the attributes this way.